web/client/components/charts/__tests__/SimpleChart-test.jsx

```diff
@@ -74,4 +74,113 @@ describe('SimpleChart component', () => {
         expect(tooltip).toExist();
 
     });
+    it('test with y axis vlaues over trillion ', () => {
+        let modifiedData = data;
+        modifiedData[6].uv = 10000000000000;
```

Comment:
This is not a copy (modifyedData and data will be the same). This does what you want
```
const modifyedData = [...data];
modifiedData[6] = {...modifiedData[6], uv: 10000000000000};
```